runtime/doc/pi_health.txt

```diff
@@ -0,0 +1,73 @@
+*pi_health.txt*   Check the status of your neovim system
+
+Author: TJ DeVries <devries.timothyj@gmail.com>
+
+==============================================================================
+1. Contents					 *health.vim-contents*
+
+	1. Contents ..: |health.vim-contents|
+	2. Health.vim introduction ..: |health.vim-intro|
+	3. Health.vim manual ..: |health.vim-manual|
+		3.1 Health.vim commands ..: |health.vim-commands|
+
+==============================================================================
+2. Health.vim introduction			*health.vim-intro*
+
+Debugging common issues is a time consuming task that many developers would
+like to eliminate, and where elimination is impossible, minimize. Many common
+questions and difficulties could be answered by a simple check of an
+environment variable or a setting that the user has made. However, even with
+FAQs and other manuals, it can be difficult to suggest the path a user should
+take without knowing some information about their system.
+
+Health.vim aims to solve this problem in two ways for both core and plugin
+maintainers.
+
+The way this is done is to provide an interface for that users will know to
+check first before posting questsion in the issue tracker, dev list, etc. This
+is similar to how |:help| functions currently. The user experiencing
+difficulty can run |:CheckHealth| to view the status of her system.
+
+The aim of |:CheckHealth| is two-fold.
+
+The first aim is to provide maintainers with an overview of the state of the
```

Comment:
"overview of the state of the user's system" is a mouth full.  Could be reworded "overview of the user's system state" or "overview of the user's working environment".